python/cudf/cudf/options.py

```diff
@@ -215,9 +241,26 @@ def _integer_and_none_validator(val):
         \tValid values are True or False. Default is False.
         """
     ),
-    _make_contains_validator([False, True]),
+    _spill_validator,
 )
 
+
+_register_option(
+    "copy_on_write",
+    _env_get_bool("CUDF_COPY_ON_WRITE", False),
+    textwrap.dedent(
+        """
+        Default behavior of performing shallow copies.
+        If set to `False`, disables copy-on-write.
+        If set to `True`, enables copy-on-write.
+        Read more at: :ref: copy-on-write-user-doc
```

Comment:
Does this reference actually end up rendered somewhere?

Reply:
Would need, I think, at least ```:ref:`copy-on-write-user-doc` ```

Reply:
Fixed it. We currently don't have a doc-rendered page for options.